script.js

```diff
@@ -23,3 +23,20 @@ links.forEach((link) => {
     hideMenu();
   });
 });
+
+// Contact Form
+const form = document.getElementById('myForm');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const email = document.querySelector('.email-input').value;
+  const suggestedEmail = document.querySelector('.email-input').value.toLowerCase();
```

Comment:
-  [optional] I would suggest that instead of getting the email field value twice, you can get it once and compare it as below. This improves the efficiency of the code 📓 
```
(email === email.toLowercase())
```